docs/UserGuide.adoc

```diff
@@ -179,14 +179,26 @@ After entering an attribute, the system will prompt the user input for another a
 
 Format: `add t/**TYPE**`
 
-ORGANice will prompt the user to type in each attribute, one by one.
+When the user type in the command above, a form will be displayed. The form should looks like this:
```

Comment:
You might want to run through an example. A possible change to this sentence would be "For example, if a user wants to add a patient, type `add t/patient` to the command box. A form will be displayed, as shown in (refer to the figure name)"